src/components/shared/forms/styles/formControl.scss

```diff
@@ -25,8 +25,8 @@ $textareaHeight: 60px;
     &:disabled {
       -webkit-touch-callout: none;
         -webkit-user-select: none;
-        -khtml-user-select: none;
-          -moz-user-select: none;
+         -khtml-user-select: none;
+           -moz-user-select: none;
```

Comment:
is it the style to leave one extra space in each line?

Reply:
nope, fixed